components/dashboard/src/settings/Integrations.tsx

```diff
@@ -4,20 +4,24 @@
  * See License-AGPL.txt in the project root for license information.
  */
 
-import { AuthProviderInfo } from "@gitpod/gitpod-protocol";
-import React, { useContext, useEffect, useState } from "react";
+import * as images from '../images';
```

Comment:
Look like you are importing all images here. Is that so?

Reply:
good point!